anomalib/models/components/freia/modules/base.py

```diff
@@ -0,0 +1,113 @@
+"""Base Module."""
+
+# Copyright (c) 2018-2022 Lynton Ardizzone, Visual Learning Lab Heidelberg.
+# SPDX-License-Identifier: MIT
+#
+
+# flake8: noqa
+# pylint: skip-file
+# type: ignore
+# pydocstyle: noqa
```

Comment:
In `all_in_one_block.py`, these are at the very top of the file, above the first docstring. For consistency it would be better to move them to the top of the file here as well.

Reply:
Done.